README.md

```diff
@@ -1,2 +1,237 @@
-# booking-app-back-end
-Final Capstone Project to build a booking application using  ruby on rails as a back-end stack  
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [� Table of Contents](#-table-of-contents)
+- [📖 \[booking-app \] ](#-booking-app--)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [Kanban Board ](#kanban-board-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [booking-app ] <a name="about-project"></a>
+
+**[booking-app-bak-end]** is an application which allows users to to book/rent vehicles for the stipulated amount of time.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+<summary>Server</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Authenicate user]**
+- **[Fetch bookings]**
+- **[Post bookings]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](coming soon)
```

Comment:
Maybe it's better to leave this black or put a filler link. Consider deploying the dev version with link to swagger in live version

Reply:
resolved, thank you for your review